src/utils/composeStores.js

```diff
@@ -1,8 +1,17 @@
 import mapValues from '../utils/mapValues';
 import pick from '../utils/pick';
+import invariant from 'invariant';
 
 export default function composeStores(stores) {
   const finalStores = pick(stores, (val) => typeof val === 'function');
+
+  Object.keys(finalStores).forEach(key => {
+    invariant(
```

Comment:
@gaearon I guess the answer to that is: would `undefined` be a valid state? If so, then I think that we should warn, otherwise fail :)